aten/src/ATen/cuda/CUDAEvent.h

```diff
@@ -13,6 +13,16 @@
 #include <cstdint>
 #include <utility>
 
+/*
+* `cudaEventTimingOnly` is a torch-specific flag that is used to indicate that
+* the CUDAEvent will only be used for timing, and never for synchronization.
+* CUDAEvents used for intra-graph timing must be recorded with `cudaEventRecordExternal`,
```

Comment:
* CUDAEvents used for intra-graph timing must be recorded with `cudaEventRecordExternal`,
* whereas CUDAEvents used for inter-graph synchronization must never be recorded with

I believe you need to swap "intra-graph" and "inter-graph" in this comment. "intra-graph" means within the graph (i.e., the meaning in https://docs.nvidia.com/cuda/cuda-c-programming-guide/#cross-stream-dependencies-and-events)